src/test/java/no/fintlabs/role/RoleServiceTests.java

```diff
@@ -0,0 +1,158 @@
+package no.fintlabs.role;
+
+import no.fintlabs.member.Member;
+import no.fintlabs.member.MemberRepository;
+import no.fintlabs.member.MemberService;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.lang.reflect.Method;
+import java.util.*;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.willDoNothing;
+import static org.mockito.Mockito.*;
+
+import no.fintlabs.opa.model.OrgUnitType;
+@ExtendWith(MockitoExtension.class)
+public class RoleServiceTests {
+
+    @Mock
+    private RoleRepository roleRepository;
+    @Mock
+    private  MemberService memberService;
+    @InjectMocks
+    private RoleService roleService;
+    private Role role;
+    @BeforeEach
+    public void setUp() {
+        role = Role.builder()
```

Comment:
Ser ut som at role som lages her brukes bare i 1 test? Setup metoden kjøres før hver test, så tanken med metoden er å ha ting som skal gjenbrukes/gjøres før hver test. 

Du lager newRole et par steder, kanskje en privat metode for å opprette role kunne vært noe?